### PR TITLE
Removes unused music, expands music track datum to include new titles.

### PR DESCRIPTION
**Pull request by Kurgis**
## About The Pull Request
Adds some licensing info.
Removes unused music.
Expands music track datum to include new titles.
Removes music with non-commercial licences.

## Why It's Good For The Game
Expansion of an existing code feature, refactoring is always nice.

## Changelog
:cl:
sounddel: Removed old music files that have now gone unused.
/:cl:

### CODE REVIEW

**Comment by AlexMorgan3817** [comment]
<h1>Deleting space oddity punishable by execution!</h1><hr>

File: code/game/objects/items/devices/player.dm [hunk 1]
```diff
@@ -4,16 +4,6 @@
 	var/current_track = ""
 	var/obj/item/holder = null
 	var/list/songs = list(
```
Comment (ITCups) [inline code/game/objects/items/devices/player.dm]: Does this means the item is useless now ?
If yes it should be deleted or made unobtainable

Comment (ITCups) [reply code/game/objects/items/devices/player.dm]: everything else looks fine tho

Comment (Kurgis) [reply code/game/objects/items/devices/player.dm]: Will address.

**Comment by Kurgis** [comment]
This is ready for review.

File: code/modules/client/preference_setup/loadout/lists/earwear.dm [hunk 2]
```diff
@@ -4,7 +4,3 @@
 	display_name = "earmuffs"
 	path = /obj/item/clothing/ears/earmuffs
 	sort_category = "Earwear"
-
-/datum/gear/ears/headphones
```
Comment (AlexMorgan3817) [inline code/modules/client/preference_setup/loadout/lists/earwear.dm]: They aren't play music....

Comment (Kurgis) [reply code/modules/client/preference_setup/loadout/lists/earwear.dm]: Old headphone code was removed and replaced with mp3 headphones in a previous PR, which is gutted out in this PR.

**Pull request merged**